docs/install/installEverest.md

```diff
@@ -0,0 +1,53 @@
+# Install Everest
+
+Percona Everest has two primary components:
+
+ - [Percona CLI](https://github.com/percona/percona-everest-cli), which connects Percona Everest to your Kubernetes cluster.
+- Everest application (UI for simplicity) that you can spin up with our quickstart instructions below.
+
+
+## Before you start
+
+Percona Everest assists with installing all the necessary operators and required packages, but does not currently help with spinning up a Kubernetes cluster.
+Therefore, before installing Everest, ensure that you have a publicly accessible Kubernetes cluster.
+We recommend setting one up on the Amazon Elastic Kubernetes Service (EKS) or Google Kubernetes Engine (GKE), as it may be difficult to currently spin one up on local Kubernetes installations (minikube, kind, k3d, or similar products).
```

Comment:
Since we're copying instructions form oepratros on how to spin up k8s on gke and aws maybe we could link this here?